### ISSUE TITLE
Tinted Theming github organization

### ISSUE THREAD

**Issue by JamyGolden** [mnussbaum/base16-waybar#1]
Hi @mnussbaum, [Tinted Theming](https://github.com/tinted-theming/) (previously base16-project) is a maintained and community run version of base16: https://github.com/tinted-theming/home/issues/51

A discussion around creating a base16 organization originally took place in 2016 on an issue in the original base16 repo: https://web.archive.org/web/20220603173440/https://github.com/chriskempson/base16/issues/74. The repo was subsequently nuked and most [git history](https://github.com/chriskempson/base16/commits/main/) as well as PRs and issues were destroyed (which is why I linked the web archive version of the issue).

I was wondering if you would be interested in joining the Tinted Theming org and continuing to maintain base16-waybar under that umbrella? It would also be good to get your repo hooked up, like the rest of our repos, to a weekly template builder bot using the latest the latest schemes in the [schemes repo](https://github.com/tinted-theming/schemes).

If you're interested let me know and I can send an org invite.

**Comment by laenzlinger** [comment]
I am using waybar and this this styles worked great for me. Would be great to have it regenerated, since in the last 6 years, many new themes. Would it help if I would do it an create a PR?

**Comment by JamyGolden** [comment]
We could fork the PR under Tinted Theming so we could have some control over the repo since it seems like the maintainer isn't interested in joining Tinted Theming. @laenzlinger if we did that, would you be interested in being the maintainer of the tinted-theming/base16-waybar repo?

**Comment by laenzlinger** [comment]
@JamyGolden Yes I would be willing to maintain it. I am interested, because I am using it already. I have to learn how to do it, but I am willing to do that

I ran locally the base16-builder-go against this this repo and it worked without issues.
